src/components/Combobox.tsx

```diff
@@ -36,15 +57,23 @@ const Combobox = ({
             leaveTo="opacity-0"
           >
             <HeadlessComboBox.Options className="z-50 absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black/5 focus:outline-none sm:text-sm">
-              {options?.map(option => (
+              {filteredOptions?.map(option => (
                 <HeadlessComboBox.Option
                   key={option}
                   value={option}
-                  className="relative cursor-pointer select-none py-2 pl-4 pr-4 text-gray-900 hover:bg-gray-100"
+                  className={optionClasses}
```

Comment:
what magic is this⁉️ :) 

I'm looking in https://headlessui.com/react/combobox and don't see an API that takes a function for className. Are we on a different version?

mind making it work more transparently?

Reply:
If you look at the interactive demo at the top and view "code" they use it there, it's also properly typed so I think it's transparent enough

<img width="564" alt="Screenshot 2024-01-08 at 10 26 00 AM" src="https://github.com/Agoric/dapp-inter/assets/8848650/c750bb34-2f48-47ee-84bf-89deb6521abd">

Reply:
The code I can find on that page uses a different function pattern,
```jsx
          <Combobox.Option key={person.id} value={person} as={Fragment}>
            {({ active, selected }) => (
              <li
                className={`${
                  active ? 'bg-blue-500 text-white' : 'bg-white text-black'
                }`}
```

but I am persuaded by the static typing. Thanks for the screenshot